### DISCUSSION TITLE
ADD NEW stable-diffusion-webui-forge packedge

### DISCUSSION THREAD

**Discussion by johnwickstyle** [LykosAI/StabilityMatrix#454] [Ideas]
### NEW stable-diffusion-webui-forge packedge 
Hello, 
recently i found :
https://github.com/lllyasviel/stable-diffusion-webui-forge
this new advance stablediffusion webui is way better then old one.
please provide  this stable-diffusion-webui-forge packedge so we can use it .
thank you.
![Screenshot 2024-02-12 013908](https://github.com/LykosAI/StabilityMatrix/assets/146026410/fb69b8af-f5e6-4164-ae99-34fe8deaa393)

**Comment by LadyFlames** [comment]
theres also Stable Cascade to go with this new Stable Forge UI 
https://stability.ai/news/introducing-stable-cascade 
https://github.com/Stability-AI/StableCascade

**Reply by johnwickstyle** [reply]
it is available in pinokio but the problem is this cascade want 24gb GPU memory to perform .

**Reply by EriIaz** [reply]
> it is available in pinokio but the problem is this cascade want 24gb GPU memory to perform .

That's not true. I mean, it might need 24GB in Pinokio due to bad implementation, but that's not an SC requirement.

You'll need 16GB if you're using the biggest models with no optimizations via diffusers wrapper as your backend, without offloading the inactive sections of the cascade to RAM. Since the performance is an afterthought for Diffusers, it's still relatively slow, and it's the most memory hungry way of running it by a long shot. 

An actually optimized backend will be much faster, you can also shuffle the models in cascade between RAM and VRAM without huge performance hits, and Stable Cascade has lite models too in case you need them, so I am sure even 8GB GPUs will be able to run it eventually.

**Reply by johnwickstyle** [reply]
thank you for your information , i think cascade doing some update to work fast and save time and memory in the future, i'm waiting for a big update to run it like SDXL.

**Reply by LadyFlames** [reply]
yea you will need at least 8-24gb preferably 8-16gb either 3060-3090 ti/Super or 4060-4090 ti/Super or you could go for 4070 Super-4080 Super ive been told by experts many times that both 4070 Super and 4080 Super are as good as 4090 and its TI version

**Comment by StabilityBro** [comment]
Look this --- 👉https://github.com/LykosAI/StabilityMatrix/discussions/482
